index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="eng">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Back to the Future Home</title>
+    <meta name="description" content="This is a page dedicated to one of our favorite movies, Back to the Future">
+    <link href="styles.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+   
+    <header class="header">
+        <nav class="header__nav">
+            <a class="header__logo" href="index.html"><img src="photos/logo.png"> </a>
+            <a class="header__link" href="index.html">Back to the Future</a>
+            <a class="header__link" href="part1.html">Part I</a>
+            <a class="header__link" href="part2.html">Part II</a>
+            <a class="header__link" href="part3.html">Part III</a>
+        </nav>
+
+        <section class="heroBanner">
+            <img src="photos/bttf.jpg">
+        </section>
+    </header>
+
+    <main>
+
+        <section class="animation">
+            <img src="photos/carfinal.png" class="flying" width="400px">
+            <style>
+            .flying {
+                position: absolute;
+                top: 50%;
+                left: -200px;
+                animation: fly 2s linear forwards;
+               }
+             
+               @keyframes fly {
+                0% {
+                 transform: translateX(0px);
+                }
+                100% {
+                 transform: translateX(1000px);
+                }
+               }
+            </style>
+        </section>
+
+        <section class="PARAGRAPH">
```

Comment:
I'm not seeing this class in the styles sheet either. If it is not being used it can be removed.